redis/src/main/scala/zio/redis/api/Streams.scala

```diff
@@ -48,6 +48,19 @@ trait Streams {
   final def xInfoStream(key: String): ZIO[RedisExecutor, RedisError, StreamInfo] =
     XInfoStream.run(XInfoCommand.Stream(key))
 
+  /**
+   * Returns the entire state of the stream, including entries, groups, consumers and PELs.
+   *
+   * @param key ID of the stream
+   * @param count limit the amount of stream/PEL entries that are returned (The first <count> entries are returned).
+   * @return General information about the stream stored at the specified key.
+   */
+  final def xInfoStreamFull(
+    key: String,
+    count: Option[Long] = None
+  ): ZIO[RedisExecutor, RedisError, StreamInfoWithFull.FullStreamInfo] =
```

Comment:
You'll have to write `Some` every time you have a count when calling this method. I suggest that you remove a `count` parameter from this method and write another method with the same name that has a `count` parameter of type `Long` in it. That way you wouldn't have to write `Some` every time you need to pass a count.

Reply:
ok,

Reply:
Is it OK to use option internally?(For the input to be generic.)

Reply:
Yes, it is.